helm-chart/templates/nextcloud-aio-database-persistentvolumeclaim.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   storageClassName: {{ .Values.STORAGE_CLASS }}
   {{- end }}
   accessModes:
-    - ReadWriteMany
+    - ReadWriteOnce
   resources:
     requests:
-      storage: {{ .Values.MAX_STORAGE_SIZE }}
+      storage: 1Gi
```

Comment:
another question: does kubernetes handle the volume permissions automatically or do I need to adjust them manually? E.g. can I set them to 777? I suppose this works different here in contrast to docker volumes where correct permissions are automatically taken from the image...

Reply:
I looked into https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods and can you tell me if https://github.com/nextcloud/all-in-one/pull/1852/files resolves the issue, WDYT?

Reply:
![image](https://user-images.githubusercontent.com/42591237/215291782-babc5658-e639-4995-87a8-88a63a485965.png)